CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Changed: The Missile Launcher now shows up as a Missile Tank when offworld in Metroid Dread.
```

Comment:
IMO this should go into dread
`Prime 1's Missile Launcher now shows up as a Missile Tank`.